ballerina/tests/test_utils.bal

```diff
@@ -1,70 +1,62 @@
-isolated function getExpectedPrompt(string message) returns string {
+isolated function getExpectedParameterSchema(string message) returns FunctionParameters {
     if message.startsWith("Rate this blog") {
-        return expectedPromptStringForRateBlog;
+        return expectedParamterSchemaStringForRateBlog;
     }
 
     if message.startsWith("Please rate this blog") {
-        return expectedPromptStringForRateBlog2;
+        return expectedParamterSchemaStringForRateBlog2;
     }
 
     if message.startsWith("What is 1 + 1?") {
-        return expectedPromptStringForRateBlog3;
+        return expectedParamterSchemaStringForRateBlog3;
     }
 
     if message.startsWith("Tell me") {
-        return expectedPromptStringForRateBlog4;
+        return expectedParamterSchemaStringForRateBlog4;
     }
 
     if message.startsWith("What's the output of the Ballerina code below?") {
-        return expectedPromptStringForBalProgram;
+        return expectedParamterSchemaStringForBalProgram;
     }
 
     if message.startsWith("Which country") {
-        return expectedPromptStringForCountry;
+        return expectedParamterSchemaStringForCountry;
     }
 
     if message.startsWith("Who is a popular sportsperson") {
-        return string `Who is a popular sportsperson that was born in the decade starting
-            from 1990 with Simone in their name?
-        ---
-
-        The output should be a JSON value that satisfies the following JSON schema, 
-        returned within a markdown snippet enclosed within ${"```json"} and ${"```"}
-        
-        Schema:
-        {"$schema":"https://json-schema.org/draft/2020-12/schema", "type":["object", "null"], "properties":{"firstName":{"type":"string"}, "middleName":{"type":["string", "null"]}, "lastName":{"type":"string"}, "yearOfBirth":{"type":"integer"}, "sport":{"type":"string"}}, "required":["firstName", "middleName", "lastName", "yearOfBirth", "sport"]}`;
+        return {"$schema":"https://json-schema.org/draft/2020-12/schema", "type": "object", "properties":{"result":{"type":["object", "null"], "properties":{"firstName":{"type":"string"}, "middleName":{"type":["string", "null"]}, "lastName":{"type":"string"}, "yearOfBirth":{"type":"integer"}, "sport":{"type":"string"}}, "required":["firstName", "middleName", "lastName", "yearOfBirth", "sport"]}}};
     }
 
-    return "INVALID";
+    return {};
 }
 
 isolated function getTheMockLLMResult(string message) returns string {
     if message.startsWith("Rate this blog") {
-        return "4";
+        return {result: 4}.toJsonString();
```

Comment:
I would put the exact string here. Otherwise our test will depend on toJsonString also.

Reply:
I believe its ok to use `toJsonString`, Otherwise its hard to read 
this is more readable.
Anyway we are using FromJson utils in the other places of the code

Reply:
IMO, this is not something where readability is critical, test correctness is more relevant. 

> Anyway we are using FromJson utils in the other places of the code

How is this relevant?